src/app/Poll/State.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Poll;
+
+use MyCLabs\Enum\Enum;
+
+/**
+ * @method static self DRAFT()
+ * @method static self PUBLISHED()
+ * @method static self FINISHED()
+ */
+class State extends Enum
```

Comment:
Change the name to `PollStatuses` to be more explicit

Reply:
Fixed in 185ceef